docs/docs/04-standard-library/01-cloud/queue.md

```diff
@@ -108,12 +106,21 @@ new cloud.Queue(props?: QueueProps);
 
 ---
 
-#### Methods <a name="Methods" id="Methods"></a>
+#### Preflight Methods <a name="Preflight Methods" id="Preflight Methods"></a>
 
 | **Name** | **Description** |
 | --- | --- |
 | <code><a href="#@winglang/sdk.cloud.Queue.setConsumer">setConsumer</a></code> | Create a function to consume messages from this queue. |
 
+#### Inflight Methods
+
+| **Name** | **Description** |
+| --- | --- |
+| <code><a href="#@winglang/sdk.cloud.IQueueClient.approxSize">approxSize</a></code> | Retrieve the approximate number of messages in the queue. |
+| <code><a href="#@winglang/sdk.cloud.IQueueClient.pop">pop</a></code> | Pop a message from the queue. |
+| <code><a href="#@winglang/sdk.cloud.IQueueClient.purge">purge</a></code> | Purge all of the messages in the queue. |
+| <code><a href="#@winglang/sdk.cloud.IQueueClient.push">push</a></code> | Push a message to the queue. |
+
 ---
 
 ##### `setConsumer` <a name="setConsumer" id="@winglang/sdk.cloud.Queue.setConsumer"></a>
```

Comment:
The ordering of headings here might be confusing:
```
- Preflight Methods
- Inflight Methods
  - setConsumer
  - ...
  - approxSize
  - ...
```

Would it make sense to put a single title named "Methods" and have the two tables under it?

Reply:
Another option could be to put the `preflight` or `inflight` keyword next to the method names in the table. I'm also okay with the current solution

Reply:
Great idea! Eyal and I liked that :) 
![image](https://github.com/winglang/wing/assets/39455181/ba91e486-949c-4cb6-a73e-30ac21236d10)